### PR TITLE
[PPD-171] change organization call

### PR DESCRIPTION
**Pull request by alessio-acitelli**
#### List of Changes

- added apiconfig call to check organization

#### Motivation and Context

- the call for organization check made on apiconfig guarantees better performance and more reliable verification

#### How Has This Been Tested?

Junit and Postaman test

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by alessio-acitelli** [comment]
Il task PPD-171 ha impatti su due applicativi:

- payments
- gpd

Quale label utilizzo?

File: payments/src/main/java/it/gov/pagopa/payments/service/ApiConfigClient.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import feign.FeignException;
+import it.gov.pagopa.payments.config.FeignConfig;
+import it.gov.pagopa.payments.model.creditorinstitution.StationCreditorInstitution;
+
+
+@FeignClient(value = "apiconfig", url = "${service.apiconfig.host}", configuration = FeignConfig.class)
+public interface ApiConfigClient {
+
+    @Cacheable(value = "payment-options")
```
Comment (jacopocarlini) [inline payments/src/main/java/it/gov/pagopa/payments/service/ApiConfigClient.java:18]: use a relevant value

File: payments/src/test/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidatorTest.java [hunk 2]
```diff
@@ -1,40 +1,79 @@
...
+
+	@Test
+	void isAuthorize_OK() {
+		when(apiConfigClient.getOrganization(anyString(),anyString())).thenReturn(new StationCreditorInstitution());
+		
+		ReflectionTestUtils.setField(paymentValidator, "ptIdIntermediario", "bbb");
+		ReflectionTestUtils.setField(paymentValidator, "ptIdStazione", "ccc");
+
+		paymentValidator.isAuthorize("aaa", "bbb", "ccc");
+		// se arrivo qui la chiamata è andata a buon fine
```
Comment (jacopocarlini) [inline payments/src/test/java/it/gov/pagopa/payments/endpoints/validation/PaymentValidatorTest.java:44]: use english

**Review by jacopocarlini — CHANGES_REQUESTED** [review]

**Review by jacopocarlini — APPROVED** [review]
Good 🔝

**Pull request merged**